src/uwtools/drivers/sfc_climo_gen.py

```diff
@@ -0,0 +1,162 @@
+"""
+A driver for sfc_climo_gen.
+"""
```

Comment:
This follows the pattern set by the FV3 driver: Instantiate the driver object, which entails reading in and validating the config file against its schema, then provide graph-task methods that can be called from the CLI or API.